CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing
+
+Welcome, and thank you for your interest in contributing to the AL developer tools! The goal of this document is to provide a high-level overview of how you can get involved.
+
+## Asking questions
+
+Have a question? Rather than opening an issue, please use one of the following resources:
+
+- [Business Central Community](https://community.dynamics.com/business/f/758)  
+- [mibuso forum](https://forum.mibuso.com/categories/nav-three-tier)  
+- [Dynamics User Group](https://dynamicsuser.net/nav/f/developers)  
+  
+The community will be eager to assist you and your well-worded question will serve as a resource to others searching for help.
+
+## Reporting issues
```

Comment:
Any risk of reading this as being about AL report issues? Might just be me (old reporting PM), alternatively we can write "Logging issues"